### PR TITLE
Use Hyperion for persistence.

### PR DESCRIPTION
**Pull request by robyurkowski**

### CODE REVIEW

**Comment by parndt** [comment]
So this replaces #51

File: core/spec/entities/content/text_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+require 'abc/entities/content/text'
+
+module Abc
+  module Entities
+    module Content
+      describe Text do
+        let(:text) { rand(30).times.map{|i| ('A'..'Z').to_a.sample}.join }
```
Comment (ugisozols) [inline core/spec/entities/content/text_spec.rb]: Why this complexity when we don't care about text value?

**Comment by knewter** [comment]
:heart:

File: adapters/abc-adapters.gemspec [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  s.description = Abc::DESCRIPTION
+
+  s.require_paths = ["lib"]
+
+  s.files = `git ls-files`.split($/)
+  s.test_files = s.files.grep(%r{^(spec)/})
+
+  s.required_ruby_version = '>= 1.9.3' # that's right.
+
+  s.add_dependency 'abc', "~>#{Abc::VERSION}"
```
Comment (robyurkowski) [inline adapters/abc-adapters.gemspec:21]: This should be a testing dependency on core I think.

File: adapters/lib/abc-adapters.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+$LOAD_PATH.push File.expand_path('../../abc', __FILE__)
+require 'adapters/config'
+require 'adapters'
```
Comment (robyurkowski) [inline adapters/lib/abc-adapters.rb:3]: All the relative requires in abc/adapters should go in adapters/lib/abc-adapters so we don't need this require relative stuff here.

**Pull request merged**